src/sentry/api/serializers/models/team.py

```diff
@@ -19,6 +19,7 @@
     TeamAvatar,
     User,
 )
+from sentry.scim.endpoints.constants import SCIM_SCHEMA_GROUP  # type: ignore
```

Comment:
not sure why this import was throwing an import error, so ignored it. was getting `rror: Skipping analyzing 'sentry.scim.endpoints.constants': found module but no type hints or library stubs`, @getsentry/python-typing any ideas? i thought we had follow imports turned off.

Reply:
No idea tbh. I'd just type scim.endpoints.constants, does not seem like a lot of work.

Reply:
even after typing it still threw an error. turned out to be because of needing `__init__.py` at each level of the folder structure of the module. https://stackoverflow.com/questions/60873894/mypy-cant-find-submodule

Reply:
@flub just hit the same problem, yes